### PR TITLE
Update stp.md

### LINKED ISSUE #120: Add more STP information
There's lots of new information available about the curriculum and information for trainers, which I think would be useful to add.

### PR DESCRIPTION
**Pull request by hshuaib90**
Fixes #120

### CODE REVIEW

File: _pages/stp.md [hunk 1]
```diff
@@ -1,49 +1,108 @@
 ---
+2
```
Comment (heyhaleema) [inline _pages/stp.md]: It seems like this is a line number?

File: _pages/stp.md [hunk 2]
```diff
@@ -1,49 +1,108 @@
...
+42
+​
+## Trainers
 
+The STP [Trainers](https://nshcs.hee.nhs.uk/programmes/stp/trainers/) section of the NSHCS website is kept up to date with new information and resources.
+ 
+We’ve covered all of the essentials on working with the 2022 curriculum, this includes:  
+·       changes to [rotations](https://nshcs.hee.nhs.uk/programmes/stp/trainers/guidance-for-working-with-the-2022-stp-curricula/rotation-modules/) which now provide trainees with an understanding of their specialty and the wider healthcare environment around them
```
Comment (heyhaleema) [inline _pages/stp.md]: This bullet point format doesn't seem to render well on the page/match the previous bullet point formatting:

``` suggestion
-  changes to [rotations](https://nshcs.hee.nhs.uk/programmes/stp/trainers/guidance-for-working-with-the-2022-stp-curricula/rotation-modules/) which now provide trainees with an understanding of their specialty and the wider healthcare environment around them,
```

File: _pages/stp.md [hunk 3]
```diff
@@ -1,49 +1,108 @@
...
+42
+​
+## Trainers
 
+The STP [Trainers](https://nshcs.hee.nhs.uk/programmes/stp/trainers/) section of the NSHCS website is kept up to date with new information and resources.
+ 
+We’ve covered all of the essentials on working with the 2022 curriculum, this includes:  
+·       changes to [rotations](https://nshcs.hee.nhs.uk/programmes/stp/trainers/guidance-for-working-with-the-2022-stp-curricula/rotation-modules/) which now provide trainees with an understanding of their specialty and the wider healthcare environment around them  
+·       planning and [scheduling training](https://nshcs.hee.nhs.uk/programmes/stp/trainers/guidance-for-working-with-the-2022-stp-curricula/scheduling-training/) through the phases of the curriculum,  
+·       the [work-based curriculum](https://nshcs.hee.nhs.uk/programmes/stp/trainers/guidance-for-working-with-the-2022-stp-curricula/the-work-based-curriculum/) including the updated competencies and considerations, work-based assessments and clinical experiences
```
Comment (heyhaleema) [inline _pages/stp.md]:
```suggestion
- the [work-based curriculum](https://nshcs.hee.nhs.uk/programmes/stp/trainers/guidance-for-working-with-the-2022-stp-curricula/the-work-based-curriculum/) including the updated competencies and considerations, work-based assessments and clinical experiences,
```

File: _pages/stp.md [hunk 4]
```diff
@@ -1,49 +1,108 @@
...
+·       changes to [rotations](https://nshcs.hee.nhs.uk/programmes/stp/trainers/guidance-for-working-with-the-2022-stp-curricula/rotation-modules/) which now provide trainees with an understanding of their specialty and the wider healthcare environment around them  
+·       planning and [scheduling training](https://nshcs.hee.nhs.uk/programmes/stp/trainers/guidance-for-working-with-the-2022-stp-curricula/scheduling-training/) through the phases of the curriculum,  
+·       the [work-based curriculum](https://nshcs.hee.nhs.uk/programmes/stp/trainers/guidance-for-working-with-the-2022-stp-curricula/the-work-based-curriculum/) including the updated competencies and considerations, work-based assessments and clinical experiences  
+·       changes to the core modules  including the new [Professional Foundations](https://curriculumlibrary.nshcs.org.uk/stp/module/S-C1/) and [Independent Professional Development](https://curriculumlibrary.nshcs.org.uk/stp/module/S-C4/) modules. We’d suggest starting to look at both of these modules with trainees as they start, there’s lots to consider as they begin to develop their professional skills and knowledge.
```
Comment (heyhaleema) [inline _pages/stp.md]:
```suggestion
- changes to the core modules including the new [Professional Foundations](https://curriculumlibrary.nshcs.org.uk/stp/module/S-C1/) and [Independent Professional Development](https://curriculumlibrary.nshcs.org.uk/stp/module/S-C4/) modules. We’d suggest starting to look at both of these modules with trainees as they start, there’s lots to consider as they begin to develop their professional skills and knowledge. 
```

File: _pages/stp.md [hunk 5]
```diff
@@ -1,49 +1,108 @@
...
+6
 ---
-
+7
+​
+8
 The NHS employs Clinical Scientists who work in scientific roles within hospitals. The NHS Scientist Training Programme 
+9
 (STP) is a three-year [scheme](https://nshcs.hee.nhs.uk/programmes/stp/) of work-based learning, supported by a 
+10
```
Comment (heyhaleema) [inline _pages/stp.md]: Weirdly, there were [zero width-spaces](https://en.wikipedia.org/wiki/Zero-width_space) here and in other lines - maybe related to the formatting/line numbers? I don't think it actually affects how the page looks, though.

image

**Review by heyhaleema — CHANGES_REQUESTED** [review]
Looks 👌 ! I've left a few comments for some minor formatting edits before this gets merged.

**Comment by hshuaib90** [comment]
Sorry @heyhaleema idk what happened with the formatting and numbers, it was horrendous. I think a copy-paste job gone wrong! I should've been more diligent, sorry. Hopefully, now it's fine.

**Review by heyhaleema — APPROVED** [review]
No worries! New content is 🔥

**Pull request merged**